### PR TITLE
Feature/about me page

### PR DESCRIPTION
**Pull request by KyleSaunders1310**

### CODE REVIEW

File: src/app/components/about-me-page/about-me-page.component.ts [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
-
+import { Component, ElementRef, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { CarouselModule } from 'ngx-owl-carousel-o';
 @Component({
   selector: 'app-about-me-page',
   templateUrl: './about-me-page.component.html',
   styleUrls: ['./about-me-page.component.scss']
 })
 export class AboutMePageComponent implements OnInit {
+  @ViewChild('journeyItems') journeyItemsRef: any
```
Comment (Solenoden) [inline src/app/components/about-me-page/about-me-page.component.ts:9]: Is this being used?

File: src/app/components/about-me-page/about-me-page.component.ts [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
+  public journeyCurrentIndex = 1;
 
   constructor() { }
 
   ngOnInit(): void {
   }
 
+  journeyNavigatePrevious() {
```
Comment (Solenoden) [inline src/app/components/about-me-page/about-me-page.component.ts:17]: You should set the access scope and return type on your functions

File: src/app/components/footer/footer.component.scss [hunk 3]
```diff
@@ -88,3 +88,6 @@
     fill: #FFFFFF;
     transition: all 0.5s ease;
 }
+
+:host {
```
Comment (Solenoden) [inline src/app/components/footer/footer.component.scss:92]: Empty styling

File: src/app/components/navbar/navbar.component.scss [hunk 4]
```diff
@@ -84,4 +84,8 @@ p {
     display:flex;
     align-items: center;
     align-self: center;
+}
+
+:host {
```
Comment (Solenoden) [inline src/app/components/navbar/navbar.component.scss:89]: Empty styling

File: package.json [hunk 5]
```diff
@@ -22,7 +22,9 @@
     "@angular/platform-browser": "12.2.0",
     "@angular/platform-browser-dynamic": "12.2.0",
     "@angular/router": "12.2.0",
+    "bootstrap": "^5.1.3",
     "eslint": "8.1.0",
+    "ngx-owl-carousel-o": "^6.0.0",
```
Comment (Solenoden) [inline package.json:27]: Please remove unused carousel dependency

**Review by Solenoden — CHANGES_REQUESTED** [review]

File: src/app/components/about-me-page/about-me-page.component.ts (hunk 1 above)
Comment (KyleSaunders1310) [reply src/app/components/about-me-page/about-me-page.component.ts:9]: Removed it

File: src/app/components/about-me-page/about-me-page.component.ts (hunk 2 above)
Comment (KyleSaunders1310) [reply src/app/components/about-me-page/about-me-page.component.ts:17]: Done

File: src/app/components/footer/footer.component.scss (hunk 3 above)
Comment (KyleSaunders1310) [reply src/app/components/footer/footer.component.scss:92]: Removed it

File: src/app/components/navbar/navbar.component.scss (hunk 4 above)
Comment (KyleSaunders1310) [reply src/app/components/navbar/navbar.component.scss:89]: Removed it

File: package.json (hunk 5 above)
Comment (KyleSaunders1310) [reply package.json]: Removed it

**Review by Solenoden — APPROVED** [review]

**Pull request merged**